MUXSDKStats/MUXSDKStats/MUXSDKPlayerBinding.m

```diff
@@ -194,20 +194,18 @@ - (void) handleRenditionChange:(NSNotification *) notif {
 - (void) handleRenditionChangeInAccessLog:(AVPlayerItemAccessLog *) log {
     AVPlayerItemAccessLogEvent *lastEvent = log.events.lastObject;
     float advertisedBitrate = lastEvent.indicatedBitrate;
-    BOOL bitrateHasChanged = ![self doubleValueIsEqual:@(_lastAdvertisedBitrate) toOther:@(advertisedBitrate)];
-    BOOL isStartingPlayback = [self doubleValueIsEqual:@(_lastAdvertisedBitrate) toOther:@(0)];
-
-    if (bitrateHasChanged) {
-        if(isStartingPlayback) {
-            // This is not a renditionchange but the player playing the first rendition.
-            _lastAdvertisedBitrate = advertisedBitrate;
-            return;
-        }
-        NSLog(@"MUXSDK-INFO - Switch advertised bitrate from: %f to: %f", _lastAdvertisedBitrate, advertisedBitrate);
-        [[NSNotificationCenter defaultCenter] postNotificationName:RenditionChangeNotification object: @{
-            RenditionChangeNotificationInfoAdvertisedBitrate: @(advertisedBitrate)
-        }];
+    if (advertisedBitrate <= 0 && !_started) {
```

Comment:
could this have a negative value? Shouldn't it just be the same comparison we were doing for the first: `[self doubleValueIsEqual:@(_lastAdvertisedBitrate) toOther:@(0)];` If we are doing early exit shouldn't it be different than instead of less than? I don't think this can take negative values.

Reply:
Yes I change it to ==, so if the value is 0 and the playback hasn't started we assing the first advertisedBitrate and the early exit can happen